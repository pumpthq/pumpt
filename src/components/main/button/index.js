import Button from './button'
import CustomButton from './custom'

export default Button
export { CustomButton }
