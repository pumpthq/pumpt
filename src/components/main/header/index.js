import Header from './header'
import HeaderMini from './headerMini'
import HeaderMedium from './headerMedium'
import HeaderFull from './headerFull'

import HeaderDropDownMenu from './dropdownMenu'
import HeaderDropDownItem from './dropdownItem'

export {
    HeaderMini,
    HeaderMedium,
    HeaderFull,
    HeaderDropDownMenu,
    HeaderDropDownItem
}

export default Header
