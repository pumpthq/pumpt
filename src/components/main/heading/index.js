import H1 from './h1';
import H2 from './h2';
import H3 from './h3';

export {
    H1,
    H2,
    H3,
};
