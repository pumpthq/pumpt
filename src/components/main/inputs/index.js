import {SimpleInput} from './simple'

export {
    SimpleInput
}
