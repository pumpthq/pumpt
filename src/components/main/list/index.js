import List from './list'
import ListLink from './listLink'
import ListItem from './listItem'
import ListAccordion from './accordion'

export {
    ListLink,
    ListItem,
    ListAccordion
}

export default List
