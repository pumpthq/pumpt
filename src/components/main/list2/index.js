import List2 from './list'
import {
    List2Item,
    LIST_ITEM_TYPE_TEXT,
    LIST_ITEM_TYPE_USER_ENTERED
} from './listItem'

export {
    List2 as List,
    List2Item as ListItem,
    LIST_ITEM_TYPE_TEXT as LIST_ITEM_TYPE_TEXT,
    LIST_ITEM_TYPE_USER_ENTERED as LIST_ITEM_TYPE_USER_ENTERED
}
