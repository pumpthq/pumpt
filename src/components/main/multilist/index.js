import MultiselectComponent from './multi'
import {
    MultiItem,
    MULTI_ITEM_TYPE_TEXT,
    MULTI_ITEM_TYPE_USER_ENTERED,
    MULTI_ITEM_TYPE_GROUP
} from './multiItem'
import { MultiItemGroup } from './multiItemGroup'

export {
    MultiselectComponent as Multi,
    MultiItem as MultiItem,
    MULTI_ITEM_TYPE_TEXT as MULTI_ITEM_TYPE_TEXT,
    MULTI_ITEM_TYPE_USER_ENTERED as MULTI_ITEM_TYPE_USER_ENTERED,
    MULTI_ITEM_TYPE_GROUP as MULTI_ITEM_TYPE_GROUP,
    MultiItemGroup as MultiItemGroup
}
