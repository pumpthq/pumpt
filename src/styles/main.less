// imported
@import "base/material.min.css";
@import "base/grid.css";

// base
@import "base/vars";
@import "base/mixins";
@import "base/helpers";
@import "base/fonts";
@import "base/common";
@import "base/layout";

// blocks
@import "blocks/text";
@import "blocks/icon";
@import "blocks/heading";
@import "blocks/image";
@import "blocks/card";
@import "blocks/link";
@import "blocks/button";
@import "blocks/logo";
@import "blocks/list";
@import "blocks/quote";
@import "blocks/wall";
@import "blocks/textfield";
@import "blocks/popup";
@import "blocks/profile-progress";
@import "blocks/select";
@import "blocks/slider";
@import "blocks/summary-head";
@import "blocks/dropdown";
@import "blocks/form";
@import "blocks/footer";
@import "blocks/header";
@import "blocks/navigation";
@import "blocks/tooltip";
@import "blocks/notification";
@import "blocks/checkbox";
@import "blocks/panel";
@import "blocks/column";
@import "blocks/scroll-container";
@import "blocks/swipe-container";
@import "blocks/social";
@import "blocks/figure";
@import "blocks/drawer";
@import "blocks/chat";
@import "blocks/radio";
@import "blocks/job-form";
@import "blocks/application-form";
@import "blocks/company";
@import "blocks/candidate";
