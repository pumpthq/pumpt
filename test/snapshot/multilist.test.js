import React from 'react';
import { Multi } from '../../src/components/main/multilist';
import renderer from 'react-test-renderer';

test('Multilist renders properly', () => {
  const component = renderer.create(
    <Multi
      handleGroups={true}
      items={[{id: 'one', key: 'one', text: 'choice 1 parent', value: 'option 1 parent',
                items: [{id: 'two', key: 'two', text: 'choice 1 child', value: 'option 2'}]},
                {id: 'three', key: 'three', text: 'choice 2 p', value: 'option 2', items: []}]}>
    </Multi>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
