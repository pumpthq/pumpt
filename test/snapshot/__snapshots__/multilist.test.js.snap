// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Multilist renders properly 1`] = `
<ul
  className="multi"
>
  <li
    className="list__item list__item_sub_true"
  >
    <a
      className="link link_size_xl"
      onClick={[Function]}
    >
      choice 1 parent
    </a>
    <ul
      className="multi multi_type_sublayer"
    >
      <li
        className="link_size_xl multi__item link"
        onClick={[Function]}
      >
        <span
          className="link_size_xl"
        >
          choice 1 child
        </span>
      </li>
    </ul>
  </li>
  <li
    className="list__item list__item_sub_true"
  >
    <a
      className="link link_size_xl"
      onClick={[Function]}
    >
      choice 2 p
    </a>
    <ul
      className="multi multi_type_sublayer"
    />
  </li>
</ul>
`;
